src/frontend/typer/sanity/modaler/utenlandsopphold.ts

```diff
@@ -40,7 +40,7 @@ export interface IUtenlandsoppholdTekstinnhold {
     flerePerioderSporsmal: LocaleRecordString;
     knapper: {
         fjern: LocaleRecordString;
-        leggTil: LocaleRecordString;
+        leggTil: LocaleRecordBlock;
```

Comment:
Stemmer det at det er en block og en string her? 🤔

Reply:
Skal være to blocker, fikser 😊